### ISSUE TITLE
Can't run node since upgrading to v23.08.1

### ISSUE THREAD

**Issue by inpharmaticist** [ElementsProject/lightning#6826]
Trying to run via command
```
./lightningd
```
Which gets me 
```
 2023-10-29T19:59:02.033Z INFO    plugin-clnrest.py: Killing plugin: disabled itself: No module named 'flask_cors'
2023-10-29T19:59:02.047Z **BROKEN** lightningd: Refusing to irreversibly upgrade db from version 234 to 235 in non-final version v23.08.1-146-g15b1ccb (use --database-upgrade=true to override)
Refusing to irreversibly upgrade db from version 234 to 235 in non-final version v23.08.1-146-g15b1ccb (use --database-upgrade=true to override)
```

Then trying 

```
./lightningd --database-upgrade=true
```

Yields

```
 2023-10-29T20:01:26.088Z INFO    plugin-clnrest.py: Killing plugin: disabled itself: No module named 'flask_cors'
2023-10-29T20:01:26.100Z INFO    lightningd: Updating database from version 234 to 235
2023-10-29T20:01:26.100Z **BROKEN** lightningd: query failed: wallet/wallet.c:5722: SELECT rune, last_used_nsec FROM runes
query failed: wallet/wallet.c:5722: SELECT rune, last_used_nsec FROM runes
```

Running Pop!_OS v 22.04. Last successful CLN version was v23.05. Tried restoring with backup.py with no change in result.

**Comment by vincenzopalazzo** [comment]
What commit of master are you running?

**Comment by ShahanaFarooqui** [comment]
@inpharmaticist This db failure bug was fixed with PR #6781. Please pull the latest code from master, rebuild and try again.

And clnrest disabling itself is just a warning to inform that a required dependency is missing for the plugin to execute.

**Comment by inpharmaticist** [comment]
Thanks, that worked.
